### DISCUSSION TITLE
Seeing an issue with usePrepareContractWrite, InjectedProvider and iframes

### DISCUSSION THREAD

**Discussion by neokry** [wevm/wagmi#826] [Q&A]
When a component contains usePrepareContractWrite, an iframe  and a wallet connected with InjectedProvider it causes the browser to crash. Not sure if this is a wagmi related issue or something related to the injected provider. Happens even when enabled is set to false on usePrepareContractWrite config.

**Comment by rhaining** [comment]
@neokry were you ever able to find a fix or workaround to this? thanks!
